### DISCUSSION TITLE
Json schema refs

### DISCUSSION THREAD

**Discussion by robertlee2telus** [Apicurio/apicurio-registry#4851] [Q&A] [answered]
Hello,

The documentation shows the how to create references between AVRO schema artifacts.  Wondering if anyone can share the same samples for json schemas as well.

Thanks in advance

**Comment by EricWittmann** [comment]
We don't currently have this example, but creating something like it is on our roadmap.  Just so many things to do!  :)

**Comment by carlesarnal** [comment, marked as answer]
We recently added a blog post with some information in this area -> https://www.apicur.io/blog/2024/05/14/registry-dereference.

Hope it helps.
